src/styles/watch.scss

```diff
@@ -0,0 +1,51 @@
+.watch {
+  position: relative;
+  display: flex;
+  justify-content: center;
+  width: 80vmin;
+  height: 80vmin;
+  margin: auto;
+  border: 10px dotted #000;
+  border-radius: 50%;
+
+  &__center {
```

Comment:
For consistency, use the same units for width and height. Change '5vh' to '5%' or '5vw'.